src/endpoints/NotificationSubscriptions/types.ts

```diff
@@ -1,5 +1,10 @@
 import type { NotificationSubscription } from '../../types';
 
+interface SubscriptionUpdate {
+    notification_type: NotificationSubscription.Type['id'];
+    is_email_subscribed: NotificationSubscription['is_email_required'];
```

Comment:
Here it should be `is_email_subscribed`.

Reply:
Oh snap! Autocomplete played some nasty tricks on me :D
Thanks for the catch!